### PR TITLE
Feature/insert id fix

### PR DESCRIPTION
**Pull request by JackyKoh**

### CODE REVIEW

**Comment by boba-and-beer** [comment]
Just to confirm I understand this PR correctly - you want to support inserting and allow users to specify their`id_col`?

File: relevanceai/_api/endpoints/api_client.py [hunk 1]
```diff
@@ -83,9 +83,7 @@ def _convert_id_to_string(self, documents, create_id: bool = False):
             )
         except KeyError:
             if create_id:
-                self.set_field_across_documents(
-                    "_id", [make_id(document) for document in documents], documents
-                )
+                pass
```
Comment (boba-and-beer) [inline relevanceai/_api/endpoints/api_client.py:86]: Why was this removed? 
If this was because of the ID hashing, we should instead switch over to UUID.

**Review by boba-and-beer — APPROVED** [review]

File: relevanceai/_api/endpoints/api_client.py (hunk 1 above)
Comment (JackyKoh) [reply relevanceai/_api/endpoints/api_client.py:86]: the API handles ID creation if you dont set any.
